test/gurka/test_gtfs.cc

```diff
@@ -0,0 +1,149 @@
+#include "gurka.h"
+#include "third_party/just_gtfs/include/just_gtfs/just_gtfs.h"
```

Comment:
this should work if the cmake changes are correct (need to make changes in cmakelists in tests dir)
```suggestion
#include "just_gtfs/just_gtfs.h"
```